source/clear-linux/concepts/swupd-about.rst

```diff
@@ -101,9 +88,10 @@ And reenabled by running::
 
     # swupd autoupdate --enable
 
+To learn how to run an update of your system, visit our :ref:`Use swupd <swupd-guide>` page.
```

Comment:
For rendering the page title in Sphinx, simply use: 

```python
:ref:`swupd-guide`
```

This way of using the `:ref:` directive automatically shows the page's title.